AUTHORS

```diff
@@ -8,60 +8,61 @@ Ex-Maintainers:
     Sven Neumann      <sven at convergence dot de>                   |  neo23
 
 Committers:
-    Robert Collins        (subunit support)                          |  rbcollins
```

Comment:
> But can you explain why is that necessary?

It help with code archeology if commits that change only space are separate from commits that change code. In this case it is not that important. However, in general I try to have space-only changes in their own commits.

> Wouldn't it be better to split authors and their role into different lines? Role could be indented relative to an author.

If you would like to make this change, that would be fine. It is your call.